### DISCUSSION TITLE
🚀 Just Launched: GitBook for GitHub Copilot

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#749] [Announcements]
![GitHub Copilot chat](https://github.com/user-attachments/assets/c7e9f1a8-b92f-438b-b53d-5921ea97c540)

We're excited to announce the launch of [GitBook for GitHub Copilot](https://github.com/marketplace/gitbook-for-github-copilot)! 🎉

This new integration allows you to seamlessly leverage your GitBook documentation directly within GitHub Copilot, providing instant, personalized answers to your queries right where you work. Whether you're asking about product details, code specifics, or anything else you have documented, GitBook Copilot has you covered.

You can start using it right away by downloading it [here](https://github.com/marketplace/gitbook-for-github-copilot).

Let us know what you think, and feel free to share any feedback!

**Comment by rickysullivan-gallagher** [comment]
I've connected my GitHub and GitBook accounts, setup the connection to the space I want to use and I still can't seem to `@gitbook-copilot` in my VS Code extension. I've restarted VS Code, logged out/in. Still nothing.

<img width="727" alt="image" src="https://github.com/user-attachments/assets/1f54d3c9-1474-4048-a433-304847f8bf6e">

**Reply by robertkern** [reply]
I'm experiencing the same

**Reply by addisonschultz** [reply]
Just to make sure - the app is installed inside of GitHub as well, right? Are you able to try the copilot commands inside of GitHub as well? (https://github.com/copilot)

**Reply by rickysullivan-gallagher** [reply]
<img width="620" alt="image" src="https://github.com/user-attachments/assets/c84eccee-1242-4583-a0db-4d60b49ea678">
<img width="497" alt="image" src="https://github.com/user-attachments/assets/c3ab7c61-2dd6-4e80-a8ee-56e2b91fa3e1">
<img width="1010" alt="image" src="https://github.com/user-attachments/assets/042e8cb6-b2fb-4fca-b717-218379bf74f9">


Perhaps because it's been added to an org?

**Reply by addisonschultz** [reply]
Can you try with your personal account (in GitHub)

**Reply by rickysullivan-gallagher** [reply]
<img width="1300" alt="image" src="https://github.com/user-attachments/assets/9604cc14-0ed8-464f-8f9e-6428c7f2738a">

<img width="710" alt="image" src="https://github.com/user-attachments/assets/5b361f26-851e-4b8b-a142-e51ffc35040f">

Still no dice.

**Reply by robertkern** [reply]
For me, after i re-authenticated with GitHub (clicking the Connected button from the screenshot above) and gave GitBook the additional permissions it needed for CoPilot then it appeared for me on CoPilot on github.com (and after restarting vscode it appeared there this time).

**Reply by rickysullivan-gallagher** [reply]
Un-installed from GitBook, re-connected. Still nothing.

**Reply by rickysullivan-gallagher** [reply]
I think I figured it out. We have another org that was created as a sandbox.  This org hasn't enabled copilot for my account yet. So when I do use copilot, it's permissions are via the main org. I have sent a request to the admin for access to the sandbox org.

**Reply by addisonschultz** [reply]
That sounds like it could be the culprit - please report back if that hasn't solved it!

**Comment by robertkern** [comment]
<img width="349" alt="Screenshot 2024-10-09 at 8 49 26 PM" src="https://github.com/user-attachments/assets/bd7eccdc-d6ad-433c-ade9-fc45d40e188b">

Seems to be working pretty well but something weird is it starts most/many responses with "Answer" and some weird english. Great to see this extension though thanks!

**Reply by addisonschultz** [reply]
Awesome! Will note the weird response formatting, thank you for flagging!

**Comment by getvictor** [comment]
Does GitBook Copilot work with JetBrains Copilot extension?

**Reply by addisonschultz** [reply]
Hey @getvictor that’s a good question - I’m not fully sure off the top of my head, but I imagine it should work, as long as you have the integrations correctly installed in GitBook and GitHub, and you have access to Copilot!
